community/mei-by-laws.md

```diff
@@ -73,41 +77,51 @@ The Board is self-organizing; that is, there will be no formal regulations on ho
-The elected members of the Board each are elected for a three year term. Terms on the Board are staggered so that three positions are open for election each year. Terms of office begin January 1 at 2 pm UTC.
+### 5.2 Meetings
 
-Exception to annual elections and term lengths will be made for the initial election of the Board (see Initial election of the Board) and in case a Board member resigns.
+The Board holds regular, virtual or in-person meetings, e.g. approximately every three months. As schedules and funding permit, the Board meets in person at least once per year. For every meeting, an agenda shall be published online prior to the meeting and minutes of the meeting shall be provided in a timely manner following the meeting. Depending on the availability of funding, the members of the Board may be refunded travel costs for this meeting.
+
+### 5.3 Terms & Elections
 
-Board members may resign from their commitments at any time but are expected to give adequate notice. After accepting the resignation, the Board may decide on how to proceed concerning the newly vacant position. Options should include but are not limited to holding a special election, appointing a replacement, or leaving the position vacant.
+The elected members of the Board each are elected for a three year term. Terms on the Board are staggered so that three positions are open for election each year. Terms of office begin January 1 at 2 pm UTC.
 
-In case of a special election to fill a vacancy between regular elections the newly elected Board member will serve for the remainder of the term of the vacant position.
+Exception to annual elections and term lengths will be made for the initial election of the Board and in case a Board member resigns.
 
 #### 5.3.1 Eligibility
 
 In order to be available for election to the Board, candidates must hold Personal Membership in MEI. Individuals who have previously served on the Board will be eligible for re-appointment.
 
 #### 5.3.2 Election procedure
 
-In order to determine a range of candidates, a four-week nomination phase will precede the elections, in which self-nominations are permitted. Individuals will be informed of their nomination and asked to confirm their willingness to serve on the Board determined. Elections will be held in a timely manner after determination of a final list of candidates. A ranked voting system without minimum voter turnout will be applied.
+In order to determine a range of candidates, a two-week nomination phase will precede the elections, in which self-nominations are permitted. Individuals will be informed of their nomination and asked to confirm their willingness to serve on the Board determined. Elections will be held in a timely manner after determination of a final list of candidates. Elections should be open for at least two weeks. Elections should close before the start of the terms of office of the elected candidates.
+A ranked voting system without minimum voter turnout will be applied.
```

Comment:
Suggest changing "A ranked voting system without minimum voter turnout will be applied." to 
"All open board seats shall be contested by all candidates in one election. There is no minimum voter turnout requirement. Ballots shall allow ranked voting. The winners shall be determined by a Single Transferable Vote (STV) system. The Elections Officers shall specify detailed procedures for each election, subject to consent by the Board."
This new wording codifies existing practice, but makes it explicit.  The phrase "a ranked voting system" is quite vague. It is like saying, "the staff has two flats" instead of "the key is B♭ major". The vague statement doesn't prevent the right outcome, but it doesn't require it either. A different and worse election structure, like a separate vote for each open seat, or a clumsier counting method than STV, still complies with the vague wording.
I think the MEI practice of a single election for all candidates to fill all board seats, with ranked ballots counted by STV, is a very good choice for an organisation like MEI. Since we are modifying this part of the by-laws, let's clarify this part also.

Reply:
Editorial improvement: use "method" instead of "system", that is, "Single Transferable Vote (STV) method".

Reply:
Dear @JDLH many thanks for this good proposal putting to words our current election practice. We talked about this on our today's MEI Board meeting and decided to keep it vague (but nevertheless improve it with some of your proposals) in order to keep it open for future election practice. Nevertheless we also decided to setup an internal document for the Election Officers explaining our current practice. And, if I may, I would include  in that document some of the points you make here ;-)

Reply:
@bwbohl  I think a "best practice" document for Election Officers is a great idea. Please do use some of this text if it helps. I'd be happy to review or contribute to that document. 
A word of wisdom about bylaws like these: in good times, with little conflict, it doesn't matter what exactly the bylaws say. In troubled times, when conflict is occurring, the right bylaws language can keep the situation from getting worse. But by then it is too late to fix the wrong bylaws language. So, let's hope MEI elections continue to have little conflict.